packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -1883,7 +1889,14 @@ class StackViewport extends Viewport implements IStackViewport {
     // Todo: we need to make the rotation a camera properties so that
     // we can reset it there, right now it is not possible to reset the rotation
     // without this
-    this.getVtkActiveCamera().roll(this.rotationCache);
+    if (this.initialViewUp) {
```

Comment:
if you set it in the constructor you don't need this if here, since it will always be defined